shared/rakefiles/xk_util.rake

```diff
@@ -51,3 +51,22 @@ task :rotate_secrets_key, [:encryption_key] => [:configure_serviceaccount] do |t
   Secrets.set_secrets(@secrets, rotate_secrets = true)
   Secrets.disable_non_primary_key_versions(args[:encryption_key], new_version_id)
 end
+
+# This task destroy all keys except current one for projectowner's SA.
+# All operations will be ignored in case local credentials not present or SA usage not configured.
+task :destroy_sa_keys => [:configure] do
+  sh "
+    if [ \"$TF_VAR_serviceaccount_key\" != \"\" ] && [ -f $TF_VAR_serviceaccount_key ]; then \
```

Comment:
Not sure if it matters but `-f` behaves differently from Ruby `File.file?`

>        -f file
>               True if file exists and is a regular file.

Reply:
It behaves the same way, but `File.file?` allows to make output cleaner by not spitting SA configuration commands on every run.

Reply:
It does not behave the same way with respect to links. I mentioned it in the context of [this other comment](https://github.com/gpii-ops/gpii-infra/pull/229#discussion_r241257190).